src/fat32.c

```diff
@@ -138,5 +138,5 @@ long mega65_fat32_create_contiguous_file(char* name, long size,
 
     mega65_sdcard_writesector(root_dir_sector);
 
-    return root_dir_sector + (start_cluster - 2) * 8;
+    return root_dir_sector + (long)(start_cluster - 2) * 8;
```

Comment:
why no stdint type?

Reply:
Because `root_dir_sector` is still `long`. I agree that stdint types would be better, but I'd prefer to refactor this in a separate PR, and preferable when a test is in place. Here the idea is to convert implicit casts to explicit ones.